src/current/_includes/v24.1/essential-alerts.md

```diff
@@ -0,0 +1,533 @@
+
+**Metric**
+<br>`requests.slow.raft`
+
+**Rule**
+<br>WARNING:  `requests.slow.raft` greater than `0` for `10 minutes`
+
+**Action**
+
+- Refer to [Raft]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft) and [Replication issues]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#replication-issues).
+{% endif %}
+
+## SQL 
+
+### Node not executing SQL
+
+Send an alert when a node is not executing SQL despite having connections. `sql.conns` shows the number of connections as well as the distribution, or balancing, of connections across cluster nodes. An imbalance can lead to nodes becoming overloaded.
+
+**Metric**
+<br>[`sql.conns`]({% link {{ page.version.version }}/essential-metrics-self-hosted.md %}#sql-conns)
+<br>`sql.query.count`
+
+**Rule**
+<br>Set alerts for each node:
+<br>WARNING:  `sql.conns` greater than `0` while `sql.query.count` equals `0`
+
+**Action**
+
+- Refer to [Connection Pooling]({% link {{ page.version.version }}/connection-pooling.md %}).
+
+### SQL query failure
+
+Send an alert when the query failure count exceeds a user-determined threshold based on their application's SLA.
+
+**Metric**
+<br>[`sql.failure.count`]({% link {{ page.version.version }}/essential-metrics-self-hosted.md %}#sql-failure-count)
+
+**Rule**
+<br>WARNING:  `sql.failure.count` is greater than a threshold (based on the user’s application SLA)
+
+**Action**
+
+-  Use the [**Insights** page]({% link {{ page.version.version }}/ui-insights-page.md %}) to find failed executions with their error code to troubleshoot or use application-level logs, if instrumented, to determine the cause of error.
+
+### SQL queries experiencing high latency
+
+Send an alert when the query latency exceeds a user-determined threshold based on their application’s SLA.
+
+**Metric**
+<br>[`sql.service.latency`]({% link {{ page.version.version }}/essential-metrics-self-hosted.md %}#sql-service-latency)
+<br>[`sql.conn.latency`]({% link {{ page.version.version }}/essential-metrics-self-hosted.md %}#sql-conn-latency)
+
+**Rule**
+<br>WARNING:  (p99 or p90 of `sql.service.latency` plus average of `sql.conn.latency`) is greater than a threshold (based on the user’s application SLA)
+
+**Action**
+
+- Apply the time range of the alert to the [**SQL Activity** pages]({% link {{ page.version.version }}/monitoring-and-alerting.md %}#sql-activity-pages) to investigate. Use the [**Statements** page]({% link {{ page.version.version }}/ui-statements-page.md %}) P90 Latency and P99 latency columns to correlate [statement fingerprints]({% link {{ page.version.version }}/ui-statements-page.md %}#sql-statement-fingerprints) with this alert.
+
+{% if include.deployment == 'self-hosted' %}

```

Comment:
LGTM! @mgartner do you have any thoughts/concerns on highlighting these as essential metrics to monitor for SQL?

Reply:
Hi @mgartner , when you have a chance, would you be able to review this section: [SQL queries experiencing high latency](https://deploy-preview-18537--cockroachdb-docs.netlify.app/docs/v24.2/essential-alerts-self-hosted.html#sql-queries-experiencing-high-latency)?